fhir-model/src/main/java/com/ibm/fhir/model/path/FHIRPathType.java

```diff
@@ -469,4 +487,8 @@ public static boolean isSystemType(FHIRPathType type) {
     public static boolean isMetamodelType(FHIRPathType type) {
         return METAMODEL_TYPES.contains(type);
     }
+    
+    public static boolean isJavaType(Class<?> clazz) {
+        return clazz.getName().startsWith("java");
```

Comment:
small suggestion here.... starts with`java.` may be more precise vs `javax`

Reply:
Changed to use `java.` prefix. I only care about "java" in this case, but it would work more generally if I supported both `java.` and `javax.` prefixes.